bazel/external/wamr.BUILD

```diff
@@ -12,17 +12,20 @@ filegroup(
 cmake(
     name = "wamr_lib",
     cache_entries = {
-        "WAMR_BUILD_INTERP": "1",
-        "WAMR_BUILD_FAST_INTERP": "1",
-        "WAMR_BUILD_JIT": "0",
-        "WAMR_BUILD_LAZY_JIT": "0",
-        "WAMR_BUILD_AOT": "0",
-        "WAMR_BUILD_SIMD": "0",
-        "WAMR_BUILD_MULTI_MODULE": "1",
+        "LLVM_DIR": "$EXT_BUILD_DEPS/copy_llvm_13_0_1/llvm/lib/cmake/llvm",
+        "WAMR_BUILD_INTERP": "0",
+        "WAMR_BUILD_JIT": "1",
+        "WAMR_BUILD_AOT": "1",
```

Comment:
Changing from fast interpreter to JIT/AOT is a quite disruptive change, and much more than simply adding support for the cross-thread `clone`. Could this change be split into a separate PR and discussed there, or is `wasm_module_share` / `wasm_module_obtain` not supported for builds with interpreter?

Notably, I think it's extremely valuable for us to retain the fast interpreter for (a) lightweight proxies that don't want to include LLVM, (b) Envoy Mobile, which requires interpreter, since JIT is not allowed on iOS.

Perhaps we need 2 build options? `wamr-interp` and `wamr-llvm`?

Reply:
`wasm_module_share` / `wasm_module_obtain` are also supported for builds with interpreter.

I will split *enable JIT* into a new PR later.